src/plugins/data/common/search/aggs/buckets/date_range.test.ts

```diff
@@ -74,6 +74,31 @@ describe('date_range params', () => {
     );
   };
 
+  test('produces the expected expression ast', () => {
+    const aggConfigs = getAggConfigs();
+    const dateRange = aggConfigs.aggs[0];
+    expect(dateRange.toExpressionAst()).toMatchInlineSnapshot(`
+      Object {
+        "arguments": Object {
+          "enabled": Array [
+            true,
+          ],
+          "id": Array [
+            "date_range",
+          ],
+          "ranges": Array [
+            "[{\\"from\\":\\"now-1w/w\\",\\"to\\":\\"now\\"}]",
+          ],
+          "schema": Array [
+            "buckets",
+          ],
+        },
+        "function": "aggDateRange",
+        "type": "function",
+      }
+    `);
+  });
```

Comment:
For the agg types that already had unit tests, I added one to verify the output of `toExpressionAst`